### ISSUE TITLE
Decide on suffix underscore for member variables and enforce it

### ISSUE THREAD

**Issue by jonathanlew** [UBC-Thunderbots/Software#915]
### Description of the task
This is inconsistent in the code base, for example tactic.h: done_ vs most other member variables in the repo. We should decide on a convention and make it consistent and integrate it into clang tidy

### Acceptance criteria

- [ ] Enforce suffix underscore convention
- [ ] Item B
- [ ] Item C

### Blocked By
#839

**Comment by jonathanlew** [comment]
Very low priority and unlikely we will get around to it before Robocup 2020
